tests/source_files/RandomBlockListener.php

```diff
@@ -0,0 +1,122 @@
+<?php
```

Comment:
All files in source_files/* directory (including this one) are taken straight from the doc examples and should not be reviewed.